SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTestFlowController.cpp

```diff
@@ -40,17 +40,26 @@ void ASpatialFunctionalTestFlowController::GetLifetimeReplicatedProps(TArray<FLi
 	DOREPLIFETIME(ASpatialFunctionalTestFlowController, WorkerDefinition);
 }
 
+void ASpatialFunctionalTestFlowController::BeginPlay()
+{
+	Super::BeginPlay();
+
+	if (HasAuthority())
+	{
+		// Super hack
+		 FTimerHandle Handle;
```

Comment:
Spacing looks off here - do you have the clang linter setup?

Reply:
Yes I have Clang setup, not sure why it didn't run here. Manually fixed the spacing.